core/scripts/chaincli/handler/keeper_launch.go

```diff
@@ -42,10 +42,16 @@ type startedNodeData struct {
 // 5. fund nodes if needed
 // 6. set keepers in the registry
 // 7. withdraw funds after tests are done -> TODO: wait until tests are done instead of cancel manually
-func (k *Keeper) LaunchAndTest(ctx context.Context, withdraw bool, printLogs bool, force bool) {
+func (k *Keeper) LaunchAndTest(ctx context.Context, withdraw bool, printLogs bool, force bool, bootstrap bool) {
```

Comment:
Instead of specifying each of these params as bools individually, you can assign multiple params the same type with:

```
func (k *Keeper) LaunchAndTest(ctx context.Context, withdraw, printLogs, force, bootstrap bool)
```